README.md

```diff
@@ -112,49 +113,125 @@ ebcli rest-server --trust-node
 An api collection for Postman (https://www.getpostman.com/) is provided [here](./docs/peggy.postman_collection.json) which documents some API endpoints and can be used to interact with it.
 Note: For checking account details/balance, you will need to change the cosmos addresses in the URLs, params and body to match the addresses you generated that you want to check.
 
-## Running the relayer service
+## Running the bridge locally
+
+With the application set up, you can now use Peggy by sending a lock transaction to the smart contract.
+
+### Set-up
+
+Create a .env file with environment variable LOCAL_PROVIDER - an example configuration can be found in .env.example. For running the bridge locally, you'll only need the LOCAL_PROVIDER. Environment variables MNEMONIC and INFURA_PROJECT_ID are only required for using the Ropsten testnet (see below).
```

Comment:
would be nice to add these steps for users to copypase, ie:

```bash
cd testnet-contracts/
yarn cp .env.example .env
```

Reply:
https://github.com/cosmos/peggy/pull/63/commits/46b75761ac290b4459c284cd80c6ea4b22659041, https://github.com/cosmos/peggy/pull/63/commits/3aa4a1fea4fa417de6415a6aa9488addc81cc407